editor/create_dialog.cpp

```diff
@@ -321,6 +325,9 @@ void CreateDialog::_update_search() {
 
 	TreeItem *to_select = search_box->get_text() == base_type ? root : NULL;
 
+	// Keep references to loaded scripts to avoid loading and parsing them several times
+	HashMap<String, RES> loaded_scripts;
```

Comment:
While not necessary @pouleyKetchoupp, would it be possible to have this be a variable in the class itself rather than a local variable? And then just clear it whenever the "config" method, or maybe the "popup" method, is called? That way it doesn't have to rebuild this map on every key press, but rather keeps the scripts loaded the entire time the user has it open.

Reply:
Yes, let me look into that.